cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -82,10 +82,15 @@ static void ets_printf_P(const char *str, ...) {
     }
 }
 
+#define FAKE_REASON_NONE 255
```

Comment:
Maybe silly question, but why not just a "bool swException" or something like that?

Reply:
I was thinking maybe we should differentiate between panic, abort, maybe other cases, with just one static global, i.e.: replace s_abort_called and this user reset reason with a single enum.